### PR TITLE
update monitoring doc

### PR DESCRIPTION
**Pull request by xinggao01**

### CODE REVIEW

File: monitoring_protected_audience_api_services.md [hunk 1]
```diff
@@ -1,80 +1,141 @@
...
+
+This proposal implements mechanisms to provide telemetry data using privacy-preserving technologies. Services running inside a TEE are now able to export common system and business metrics using these mechanisms. We use methods including [differential privacy](https://en.wikipedia.org/wiki/Differential_privacy) and data aggregation to maintain the privacy-preserving nature of Protected Audience API services while providing critical metrics thatwill help ad techs monitor their systems effectively.
```
Comment (dmdabbs) [inline monitoring_protected_audience_api_services.md:72]: thatwill -> that will

File: monitoring_protected_audience_api_services.md [hunk 2]
```diff
@@ -1,80 +1,141 @@
...
-- There are a subset of metrics which are sensitive and may reveal information
-  about user activity or which might give the ad tech running this server an
-  unfair advantage. These metrics are noised using Differential Privacy.
-- Other metrics do not have noise added, but are aggregated by Open telemetry
-  for performance reasons before being published.
+*   The servers have integrated with [OpenTelemetry](https://opentelemetry.io/) (OTel) so data can be exported to monitoring systems for creating dashboards, defining alerts and other uses.
+*   System metrics are supported, such as CPU resource utilization, and server specific business metrics. These metrics are predefined in the system, instrumented into the code by the privacy sandbox developers.
+*   Metrics are either non-noised or have noise added.
+*   There is a subset of metrics that are sensitive and may reveal information about user activity or which might give the ad tech running a server an unfair advantage. These metrics are noised using differential privacy.
+*   Other metrics do not have noise added, but are aggregated by OpenTelemetryfor performance reasons, before being published.
```
Comment (dmdabbs) [inline monitoring_protected_audience_api_services.md:80]: OpenTelemetryfor -> OpenTelemetry for

File: monitoring_protected_audience_api_services.md [hunk 3]
```diff
@@ -141,173 +202,504 @@ These are metrics tracked on the BFE B&A servers.
...
+More details will be added for AWS in future
+
+
+## Differential privacy and noising
+
+To uphold the privacy guarantees of the Protected Audience API, we use [differential privacy](https://github.com/google/differential-privacy) (DP) to add noise to sensitive metrics before they leave the trusted execution environment (TEE). Ad tech partners can select the specific metrics they wish to track from a list of pre-instrumented metrics. All servers within the same service will track this identical set of metrics.
+
+In DP, the level of noise added depends on several factors, notably the available privacy budget. A higher budget allows for less noise, while a lower budget necessitates more. Our system establishes a total privacy budget encompassing all tracked privacy-sensitive metrics. This budget is then allocated proportionally across the metrics an ad tech partner chooses to monitor. Consequently, tracking more sensitive metrics means each receives a smaller share of the budget, resulting in increased noise per metric.
+
+An ad tech can choose to alter how the privacy budget is distributed across different metrics. See the
```
Comment (dmdabbs) [inline monitoring_protected_audience_api_services.md:464]: Missing reference...

>See the

Comment (xinggao01) [reply monitoring_protected_audience_api_services.md]: Thanks.

File: monitoring_protected_audience_api_services.md [hunk 4]
```diff
@@ -141,173 +202,504 @@ These are metrics tracked on the BFE B&A servers.
...
+metric { name: \"m_0\" lower_bound: 1 upper_bound: 2 }
+```
+
+[code link](https://github.com/privacysandbox/data-plane-shared-libraries/blob/main/src/telemetry/flag/config.proto#L66)
+
+#### privacy_budget_weight
+
+Default value: 1
+
+All Privacy Impacting metrics split total privacy budget based on their weight. i.e. privacy_budget = total_budget\*privacy_budget_weight /total_weight.
```
Comment (akshaypundle) [inline monitoring_protected_audience_api_services.md:599]: spaces around operators (/ and *)

File: monitoring_protected_audience_api_services.md [hunk 5]
```diff
@@ -141,173 +202,504 @@ These are metrics tracked on the BFE B&A servers.
...
+metric { name: \"m_0\" privacy_budget_weight: 2 }
+```
+
+[code link](https://github.com/privacysandbox/data-plane-shared-libraries/blob/main/src/telemetry/flag/config.proto#L72)
+
+#### drop_noisy_values_probability
+
+Default value: 0.0
+
+The probability that noisy values will be turned to 0. Setting this to a higher value ensures that the noisy values will be turned to 0, but also means that some actual (non-noisy values) may also be turned to 0. Setting this to 1 means all values will be turned to 0 (eliminating all noise, but also eliminating all useful data). This value should be set within an open range of (0.0,1.0).
```
Comment (akshaypundle) [inline monitoring_protected_audience_api_services.md:614]: i think we should say what noisy values means ... as written this one is a little unclear.

**Review by akshaypundle — CHANGES_REQUESTED** [review]

**Review by akshaypundle — APPROVED** [review]

**Pull request merged**